nistats/datasets.py

```diff
@@ -13,6 +13,9 @@
 from nilearn.datasets.utils import (
     _get_dataset_dir, _fetch_files, _fetch_file, _uncompress_file)
 
+import boto3
```

Comment:
Is this dep documented / added to readme file, etc. ?

Reply:
not yet, I realize not even added to travis, have to fix it